server/publications.js

```diff
@@ -79,3 +80,17 @@ publishComposite('socialize.pendingFriendRequests', function publishFriends(opti
         ],
     };
 });
+
+Meteor.publish('socialize.hasFriendshipRequest', function(requestedUser) {
```

Comment:
I think maybe this should be `socialize.hasFriendRequest` to match the verbiage of the other publications

Reply:
Fixed